res/values-zh-rCN/yaap_strings.xml

```diff
@@ -226,7 +237,9 @@
     <!-- Connectivity check -->
     <string name="connectivity_check_title">网络连通性检查</string>
     <string name="connectivity_check_summary">选择用于网络连通性检查的 HTTP 站点</string>
-    <string name="connectivity_check_standard">默认 (Google)</string>
+    <string name="connectivity_check_standard">Google</string>
+    <string name="connectivity_check_gos">GrapheneOS</string>
+    <string name="connectivity_check_miui">MIUI</string>
     <string name="connectivity_check_disabled">禁用</string>
```

Comment:
```suggestion
    <string name="connectivity_check_standard">默认 (Google)</string>
    <string name="connectivity_check_disabled">禁用</string>
```

there's no need to translate lines that are the same as original. and lets also keep the Chinese for "default" there. No?

Reply:
sure! i forget to check original file😢